README.md

```diff
@@ -76,15 +76,20 @@ Run the following steps on your laptop or local workstation:
 2.  Build it with Maven.
 
         cd dockerflow
-        mvn package -DskipITs
+        mvn package -DskipTests
 
-3.  Run a sample workflow:
+3. Set up the DOCKERFLOW_HOME environment.
 
-        java -jar target/dockerflow*dependencies.jar \
-        --project=MY-PROJECT \
-        --workflow-file=src/test/resources/linear-graph.yaml \
-        --workspace=gs://MY-BUCKET/MY-PATH \
-        --runner=DirectPipelineRunner
+        export DOCKERFLOW_HOME=`pwd`
```

Comment:
The `cmd` notation is considered legacy. The public Google bash style guide encourages the use of $(cmd).

https://google.github.io/styleguide/shell.xml?showone=Command_Substitution#Command_Substitution